app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```

Comment:
Research and apply "i18n" into project for translate string (using display on UI for end user)
https://guides.rubyonrails.org/i18n.html#providing-translations-for-internationalized-strings

```suggestion
    <title>t("title")</title>
```
```
# config/locales/en.yml
en:
  title: Sample App
```